pkg/scheduler/admission/podnodeconstraints/admission.go

```diff
@@ -69,9 +69,12 @@ var resourcesToCheck = map[unversioned.GroupResource]unversioned.GroupKind{
 	kapi.Resource("podtemplates"):                               kapi.Kind("PodTemplate"),
 	kapi.Resource("replicationcontrollers"):                     kapi.Kind("ReplicationController"),
 	batch.Resource("jobs"):                                      batch.Kind("Job"),
+	batch.Resource("jobtemplates"):                              batch.Kind("JobTemplate"),
+	batch.Resource("scheduledjobs"):                             batch.Kind("ScheduledJob"),
 	extensions.Resource("deployments"):                          extensions.Kind("Deployment"),
 	extensions.Resource("replicasets"):                          extensions.Kind("ReplicaSet"),
 	extensions.Resource("jobs"):                                 extensions.Kind("Job"),
+	extensions.Resource("jobtemplates"):                         extensions.Kind("JobTemplate"),
```

Comment:
`JobTemplate` exists only in `batch`, it was done as part of ScheduledJobs and is only in `batch/v2alpha1`.